src/style.css

```diff
@@ -1 +1,46 @@
 /* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+  padding: 0 50px;
+}
+
+.nav_list {
+  display: flex;
+  list-style: none;
+  padding-left: 0;
+}
+
+.is-active {
+  color: #000;
+  padding: 10px;
```

Comment:
remove the padding on link, use margin-right on each li to make a space between them